core-common/src/main/java/org/zalando/nakadi/exceptions/runtime/InvalidResourceAnnotationException.java

```diff
@@ -0,0 +1,8 @@
+package org.zalando.nakadi.exceptions.runtime;
+
+public class InvalidResourceAnnotationException extends NakadiBaseException {
+
+    public InvalidResourceAnnotationException(final String message) {
+        super("InvalidResourceAnnotation: " + message);
```

Comment:
```suggestion
        super(message);
```

Reply:
The reason for this prefix is to propagate this to the response as the exception names are not propagated. 
example:
```
{
  "title": "Unprocessable Entity",
  "status": 422,
  "detail": "InvalidResourceAnnotation: Error validating <test-annotation-:test-value>; Key name should start and end with [a-zA-Z0-9]."
}
```